esi_leap/tests/api/controllers/v1/test_lease.py

```diff
@@ -437,6 +449,34 @@ def test_get_resource_filter_default_resource_type(self, mock_get_all,
         mock_gnl.assert_called_once()
         self.assertEqual(2, mock_lgdwai.call_count)
 
+    @mock.patch('esi_leap.api.controllers.v1.utils.'
+                'check_lease_policy_and_retrieve')
+    @mock.patch('esi_leap.objects.lease.Lease.cancel')
+    def test_lease_delete(self,  mock_cancel, mock_clpar):
+        mock_clpar.return_value = self.test_lease
+
+        self.delete_json('/leases/' + self.test_lease.uuid)
+
+        mock_clpar.assert_called_once_with(self.context,
+                                           'esi_leap:lease:get',
+                                           self.test_lease.uuid,
+                                           statuses.LEASE_CAN_DELETE)
+        mock_cancel.assert_called_once()
+
+    @mock.patch('esi_leap.api.controllers.v1.utils.'
+                'check_lease_policy_and_retrieve')
+    @mock.patch('esi_leap.objects.lease.Lease.cancel')
+    def test_lease_delete_2(self, mock_cancel, mock_clpar):
```

Comment:
We don't actually need this; the delete method has no branches, and test_lease_delete tests everything